cl/opinion_page/utils.py

```diff
@@ -211,6 +232,187 @@ class RelatedCitingResults:
     timeout: bool = False
 
 
+@dataclass
+class RelatedClusterResults:
+    related_clusters: list[OpinionClusterDocument] = field(
+        default_factory=list
+    )
+    sub_opinion_pks: list[int] = field(default_factory=list)
+    url_search_params: dict[str, str] = field(default_factory=dict)
+    timeout: bool = False
+
+
+async def es_get_related_clusters_with_cache(
+    cluster: OpinionCluster,
+    request: HttpRequest,
+) -> RelatedClusterResults:
+    """Elastic Related Clusters Search or Cache
+
+    :param cluster:The cluster to use
+    :param request:The user request
+    :return:Related Cluster Data
+    """
+    cache = caches["db_cache"]
```

Comment:
is there a reason to use database caching over Redis?

Reply:
I was attempting to be consistent with other elastic cached results

Reply:
I tend to do DB caching for larger objects, myself, but it's a fun fact that Django only evicts DB cache entries when you try to access them again later. If you cache something today, it expires, and you never visit it again, it'll take up storage forever. One day we should make a script for Django itself that cleans this up.